staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

```diff
@@ -253,11 +253,11 @@ type ValidationRule struct {
 	MessageExpression string `json:"messageExpression,omitempty" protobuf:"bytes,3,opt,name=messageExpression"`
 	// reason provides a machine-readable validation failure reason that is returned to the caller when a request fails this validation rule.
 	// The HTTP status code returned to the caller will match the reason of the reason of the first failed validation rule.
-	// The currently supported reasons are: "FieldValueInvalid", "FieldValueForbidden", "FieldValueRequired", "FieldValueDuplicate".
-	// If not set, default to use "FieldValueInvalid".
+	// The currently supported reasons are: "Invalid", "Forbidden", "NotFound", "AlreadyExists".
+	// If not set, default to use "Invalid".
 	// All future added reasons must be accepted by clients when reading this value.
 	// +optional
-	Reason *field.ErrorType `json:"reason,omitempty" protobuf:"bytes,4,opt,name=reason"`
+	Reason *metav1.StatusReason `json:"reason,omitempty" protobuf:"bytes,4,opt,name=reason"`
```

Comment:
this is *not* a status reason, it's a field error reason

Reply:
Switched to use *string